src/routes/notify/success/+page.svelte

```diff
@@ -5,8 +5,8 @@
 <div class="flex flex-col gap-8 items-center text-center mt-32">
   <h1 class="text-5xl md:text-8xl font-normal">You're signed up</h1>
   <p class="text-md md:text-2xl max-w-md md:max-w-[546px]">
-    We'll get back to you shortly via email to let you know if you got one of
-    the limited tickets.
+    We'll email you when we have more information about next year's CDE
```

Comment:
@chrifro we need new copy text for this success page also => 
![image](https://github.com/gitpod-io/CDE-Universe/assets/55068936/01febbc8-2d98-41ae-ba79-0a4d2af154b6)